Makefile

```diff
@@ -46,12 +46,12 @@ MINIMUM_CLUSTERCTL_VERSION=1.8.1
 MIN_GO_VERSION=1.23
 MINIMUM_TILT_VERSION=0.25.3
 MINIMUM_PACKER_VERSION=1.8.1
-MINIMUM_CONTROLLER_GEN_VERSION=0.10.0
+MINIMUM_CONTROLLER_GEN_VERSION=0.16.5
 MINIMUM_GH_VERSION=2.12.1
 MINIMUM_KIND_USE_VERSION=v0.20.0
 MINIMUM_ENVTEST_VERSION=1.30.3
 MINIMUM_HELM_VERSION=v3.11.3
-MINIMUM_KUSTOMIZE_VERSION=4.6.1
+MINIMUM_KUSTOMIZE_VERSION=v5.5.0
```

Comment:
Why not keep the same version format as other dependancies (i.e. without the v) ?